Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM node:argon
+FROM node:alpine
 
 # Create app directory
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 # Install app dependencies
 COPY package.json /usr/src/app/
-RUN npm install
+COPY yarn.lock /usr/src/app/
+RUN yarn install
```

Comment:
You might want to do `yarn install --production` instead, otherwise the `yarn.lock` file is not very useful

Reply:
I tihnk `yarn.lock` should be used for `yarn install` as well, no? But indeed `yarn install --production` is desired here, thanks!